internal/database/polltype.go

```diff
@@ -0,0 +1,28 @@
+package database
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/uptrace/bun"
+)
+
+const ElectionPollTypeId = 1
+const ReferendumPollTypeId = 2
+
+type PollType struct {
+	bun.BaseModel `json:"-"`
+
+	ID   int    `bun:",pk,autoincrement" json:"id"`
+	Name string `bun:",notnull" json:"name"`
+}
+
+func (p *PollType) Insert(x ...bun.IDB) error {
+	db := fromVariadic(x)
+
+	if err := db.NewInsert().Model(p).Returning("id").Scan(context.Background(), &p.ID); err != nil {
+		return fmt.Errorf("insert PollType model: %w", err)
+	}
+
+	return nil
+}
```

Comment:
This doesn't look like it's used anywhere - it might be worth removing it as to not mistakenly call it or misuse it because as far as I can tell there's no sensible use for it that wouldn't need a code change elsewhere anyway.